tools/nvqpp/nvq++.in

```diff
@@ -221,7 +221,7 @@ function show_help {
 --opt-pass <pass name>
 	Append a pass to the pass pipeline by specifying its name in the pass pipeline syntax, e.g. <dialect>.<opname>(passname).
 
---save-temps
+-save-temps
```

Comment:
This could be out-of-scope for the current PR and may be addressed with https://github.com/NVIDIA/cuda-quantum/issues/1265
But, doesn't `clang` support both, if I'm not mistaken - https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-save-temps

Reply:
Thanks for spotting that. I'll make it use both.